src/Models/HydrostaticFreeSurfaceModels/update_hydrostatic_free_surface_model_state.jl

```diff
@@ -26,26 +26,155 @@ hydrostatic pressure) to the current model state. If `callbacks` are provided (i
 they are called in the end.
 """
 update_state!(model::HydrostaticFreeSurfaceModel, callbacks=[]; compute_tendencies = true) =
-         update_state!(model, model.grid, callbacks; compute_tendencies)
+    update_state!(model, model.grid, callbacks; compute_tendencies)
+
+operation_corner_points = "default" # Choose operation_corner_points to be "default", "average", "CCW", or "CW".
```

Comment:
what does `operation_corner_points` do?

Reply:
Nothing. It isn't used anywhere else in this PR. So, I deleted line 31 defining operation_corner_points.